broker/plugins/kubejobs/plugin.py

```diff
@@ -263,37 +258,31 @@ def validate(self, data):
                     "Variable \"{}\" is missing".format(key))
             if (not isinstance(data[key], data_model[key])):
                 raise ex.BadRequestException(
-                    "\"{}\" has unexpected variable type".format(key))
+                    "\"{}\" has unexpected variable type: {}. Was expecting {}"
+                    .format(key, data[key], data_model[key]))
```

Comment:
`data[key]` returns the actual value instead of its type. you mean `type(data[key])`?